deployment/flatpak/about.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flatpak
+subsection: flatpak
+section: deployment
+description: >
+	Flatpak (formerly xdg-app) is a software utility for software deployment, package 	management and application virutalization. It provides a sandbox environment in which users 	can run applications in isolation from the rest of the system.
+---
+
+# {{page.title}}
+
+[Flatpak](https://flatpak.org/) is a software utility for software deployment, package management and application virtualization. It provides a sandbox environment in which users can run applications in isolation from the rest of the system.
+
+
+
+## Advantages of Using Flatpak
+
+* Flatpaks are distro agnostic. Developers can distribute their application on every Linux distribution simultaneously with ease.
+* Flatpaks provide consistent environments. Developers can develop and test applications in environments that are identical to the one users have.
+* Flatpak's build tools are simple and very easy to use. A link to the full documentation can be found in the "Further Reading" section below.
+* Flatpak runtimes provide stable sets of dependencies that can be used by applications.
+* Dependencies that aren't in a runtime can be bundled as part of an application. Versions of the dependencies that are used can be fixed and patched.
+* Flatpaks are future-proof. Runtimes are forward compatible, irrespective of distribution changes. Flatpak applications run on new Linux distribution versions as they are released.
```

Comment:
This highly depends on the content itself, right? Do ve have some initiative on that in Fedora? In upstream? Can this be linked to?
I cannot think of a value that this bring to a Fedora user/developer. This is a 'selling point' (vision?) and not an real advantage IMHO.